src/components/ShoppingCart/ShoppingCartContainer.tsx

```diff
@@ -53,11 +76,18 @@ const ShoppingCartContainer = ({ shoppingCart }: userCartList) => {
             {shoppingCart.map((cartRow) => (
               <CartListItem
                 key={cartRow.user_id}
-                user_id={cartRow.user_id}
-                design={cartRow.design}
-                quotation={cartRow.quotation}
-                quotationDetails={cartRow.quotationDetails}
-                previewPic={cartRow.previewPic}
+                item={cartRow}
+                onDelete={(id) => {
+                  setCartItemIdToDelete(id);
+                  onOpen();
+                }}
+                // id={cartRow.id}
+                // key={cartRow.user_id}
+                // user_id={cartRow.user_id}
+                // design={cartRow.design}
+                // quotation={cartRow.quotation}
+                // quotationDetails={cartRow.quotationDetails}
+                // previewPic={cartRow.previewPic}
```

Comment:
Don't leave anything like that. If you need to do it in the future, add "Todo" as the start of the comment.

Reply:
These are original codes from cc06 and we comment them in case they want to restore; we now understand it is not a good behavior.